gallery/others/plot_visualization_utils.py

```diff
@@ -460,3 +460,63 @@ def show(imgs):
 
 res = draw_keypoints(person_int, keypoints, connectivity=connect_skeleton, colors="blue", radius=4, width=3)
 show(res)
+
+# %%
+# That looks pretty good.
+#
+# .. _draw_keypoints_with_visibility:
```

Comment:
Nit: is this ref being used anywhere?

Reply:
It is not yet. But I thought about adding it to the parameter description of ``visibility`` in the draw_keypoints method. Seemingly I forgot. Do you think this would help? I'm not sure that the parameter explanation is sufficient for inexperienced users.